runner/runners/invoke.go

```diff
@@ -131,17 +134,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 
 	switch st.State {
 	case execer.COMPLETE:
-		srcToDest := map[string]string{
-			stdout.AsFile(): "STDOUT",
-			stderr.AsFile(): "STDERR",
+		tmp, err := temp.NewTempDir("", "invoke")
```

Comment:
right - it was my understanding that we set $TMPDIR in aurora and os.TEMPDIR() uses $TMPDIR when it is available.  So our current implementations are safe.    Just recommending that we eliminate the parameter since we really don't want it to be set to anything besides ""/os.TEMPDIR().  (I only realized just recently realized that sending in a non-"" dir would be bad.)